bin/check-all.sh

```diff
@@ -60,9 +60,9 @@ fi
 if [ -e "${base_dir}/gradlew" ]; then
   gradle_file="${base_dir}/gradlew"
 else
-  gradle_base_file=$(which gradle)
+  gradle_base_file=$(which gradle) && chk_gradle=$? || chk_gradle=$?
```

Comment:
hey @vjagadish1989  thanks for checking this out. The problem is that in my case the $(which gradle) command failed, and then the script exits without prompting anything, not even getting to the <-f> check.
So this line puts the result of executing $(which gradle) into the variable chk_gradle so we can check if the $(which gradle) didn't fail.